### DISCUSSION TITLE
Not working with commonjs since v3

### DISCUSSION THREAD

**Discussion by co-sic** [shellscape/jsx-email#77] [General]
Not sure if this is intended or not, but i'm currently trying to upgrade vom v1.4.3 of the cli and v2.2.3 of /all to v3.0.1 and now i get the following error when using ts-node to run a script or also just with node to call the render function:

```
var import_shikiji = require("shikiji");
                     ^
Error [ERR_REQUIRE_ESM]: require() of ES Module /***/node_modules/shikiji/dist/index.mjs not supported.
```

i'm using Node v 20.0.7, typescript v 5.2.2 with module options set to commonjs in my tsconfig.
Switching to ESM is not really an option for me. 

And FYI: the invite link to the discord in the Issue templates is not valid/not working.

Edit: The included dev server is working fine.

**Comment by shellscape** [comment]
This would be better off as a bug issue, with a reproduction included. I ran the version published last night through _a ton_ of testing. I typically run in a CJS env, and the CLI is CJS so while the error might have you think it's a CJS/ESM issue, it's probably not. Spin up a reproduction using the stackblitz template, and we'll figure it out.
